src/components/views/styl/Spinner.styl

```diff
@@ -7,12 +9,12 @@ main > .spinner {
     /* Give this element a static height so that the next button doesn't get
      * pushed down when the spinner appears. */
     height: 200px;
-}
 
-main > .spinner + .next-button {
-    margin-top: 0;
+    + .next-button {
```

Comment:
Doesn't this need to be `& + .next-button`?

Reply:
`&` just means trim the whitespace when setting the target. It would mean the difference between `main > .spinner + .next-button` and `main > .spinner+ .next-button`. Since we have a next sibling operator it should work in both cases but in other places we'd use it to target things with compound classes for example.

This is going off memory of using stylus years ago...I should double-check my own assertion.